plugins/inputs/nightscout/README.md

```diff
@@ -0,0 +1,1142 @@
+# Nightscout Input Plugin
+
+The nightscout plugin collects blood glucose related telemtry from a nightscout site. Not all available data is collected at this time, simply the basics needed for active monitoring.
+
+### Configuration:
+
+```toml
+  ## This is a list of nightscout sites to pull data from.
+  ##  example: servers = ["https://my.nightscout.site/api/v1/entries"]
+  servers = []
+  ## The person is the owner of the nightscout site. The order of the list of owners must match the order of the
+  ## list of servers, for example the first server should match the first owner.
+  # owners = ""
+
+  ## The secret is the SHA-1 hash of the api secret used to access the site.
+  secret = ""
+
+  ## The count is the number of results to pull at a time. Unless you have a specific need, this should remain as 1.
+  # count = "1"
+
+  ## The token is the unique token that matches a role within your nightscout site(s).
+  token = ""
```

Comment:
How does that work with multiple servers?

Reply:
Now that I am looking at it, it won't thanks for the tip. This is going to go away though in favor of simply using multiple plugin instances.

Reply:
Confirmed multiple plugin instances work as expected.